### ISSUE TITLE
safe scheduler

### ISSUE THREAD

**Issue by xlc** [open-web3-stack/open-runtime-module-library#482]
implements the scheduler traits but using OCW to trigger & unsigned tx to execute

https://github.com/paritytech/substrate/blob/master/frame/support/src/traits/schedule.rs

this ensures a bad schedule won't brick the chain

https://github.com/paritytech/substrate/issues/8740#issuecomment-833212430

**Comment by kianenigma** [comment]
But the doing this in OCW will imply totally different security guarantees. Is that acceptable? 

Why is this an issue if you already have on-idle-scheduler somewhere? 

cc @gpestana.

**Comment by xlc** [comment]
Yeah it cannot be a drop in replace of pallet-scheduler. But for many use cases, OCW should be sufficient.
We can have both pallet installed in a runtime and configure individually for each pallet on which scheduler to use.
